src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/toolbars/CustomizedToolbarComponentObject.java

```diff
@@ -38,6 +38,8 @@ public CustomizedToolbarComponentObject(WebDriver driver) {
   private WebElement resetDefaultsButton;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)